ios/Podfile.lock

```diff
@@ -253,6 +253,8 @@ PODS:
     - React-cxxreact (= 0.61.5)
     - React-jsi (= 0.61.5)
     - ReactCommon/jscallinvoker (= 0.61.5)
+  - RNCAsyncStorage (1.9.0):
```

Comment:
I don't think you need to check this file in

Reply:
this file is under version control, i thought it would be best to add
TBH, i dont know what this file is for Lol

Reply:
Ahh ok